store-conf/conf/imap.log4j.properties

```diff
@@ -0,0 +1,53 @@
+# An example log4j configuration file that outputs to System.out.  The
```

Comment:
I am OK with not adding a *properties.in file until we need to use variables in it, which AFAIK is why we have all *.in files.